utils/useBalance/config.ts

```diff
@@ -1,4 +1,4 @@
-import { AvailableWalletNames } from '@/web3-wallets/types'
+import { AvailableWalletNames } from '../../types'
```

Comment:
> Are you sure we need to refactor this file importing?

Yes, because checkout-frontend does not yet have relative imports and it will break there :(